tools/flaky-test-reporter/result.go

```diff
@@ -130,7 +130,7 @@ func flakyRateAboveThreshold(rd *RepoData) bool {
 // createArtifactForRepo marshals RepoData into json format and stores it in a json file,
 // under local artifacts directory
 func createArtifactForRepo(rd *RepoData) error {
-	outFilePath := path.Join(prow.GetLocalArtifactsDir(), rd.Config.Repo+".json")
+	outFilePath := path.Join(prow.GetLocalArtifactsDir(), rd.Config.Name+".json")
```

Comment:
can we do a structured layout here? i.e.
```
- serving
  - ci-knative-serving-continuous
  - ci-knative-serving-istio-1.0.7-mesh
  ...
```

Reply:
if you agreed please make sure it works since the directory needs to exist before creating file

Reply:
Got it, see the changes in main.go